tests/caasp/rcshell_start.pm

```diff
@@ -15,7 +15,7 @@ use strict;
 use testapi;
 
 sub run {
-    assert_screen 'startshell', 120;
+    assert_screen 'startshell', 150;
```

Comment:
I've increased that because it was failing in my testing machine (for 2 seconds only).